ratatui/src/widgets/stateful_widget_ref.rs

```diff
@@ -64,20 +67,18 @@ pub trait StatefulWidgetRef {
     fn render_ref(&self, area: Rect, buf: &mut Buffer, state: &mut Self::State);
 }
 
-// Note: while StatefulWidgetRef is marked as unstable, the blanket implementation of StatefulWidget
-// cannot be implemented as W::State is effectively pub(crate) and not accessible from outside the
-// crate. Once stabilized, this blanket implementation can be added and the specific implementations
-// on Table and List can be removed.
-//
-// /// Blanket implementation of `StatefulWidget` for `&W` where `W` implements `StatefulWidgetRef`.
-// ///
-// /// This allows you to render a stateful widget by reference.
-// impl<W: StatefulWidgetRef> StatefulWidget for &W {
-//     type State = W::State;
-//     fn render(self, area: Rect, buf: &mut Buffer, state: &mut Self::State) {
-//         StatefulWidgetRef::render_ref(self, area, buf, state);
-//     }
-// }
```

Comment:
Now that this is removed, does it mean that the widget ref is stabilized?

Reply:
Not quite - there's still some issues around committing to trait / method naming etc. Thanks for the doctest fixes btw.